core/src/main/java/io/opencensus/tags/TagValueString.java

```diff
@@ -46,6 +46,7 @@
    * @throws IllegalArgumentException if the {@code String} is not valid.
    */
   public static TagValueString create(String value) {
+    // TODO(sebright): Should we disallow empty?
```

Comment:
Thanks.  Why do you think that values shouldn't be empty?  Do you think that the same should apply to keys?  I'm not sure what we should allow, but I think tag values should be no more restrictive than tag keys, since tag keys are more likely to be known statically.